server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -52,6 +52,26 @@ public class TestSqlHostVariable {
           + "        PROCEDURE DIVISION.\n"
           + "         {_01 VAR-NAME USAGE IS SQL TYPE IS RESULT-SET-LOCATOR VARYING|1_}.";
 
+  public static final String TEXT4 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*VAR-NAME1} USAGE IS SQL TYPE IS BINARY LARGE OBJECT (10).\n"
```

Comment:
Cannot be 01 for the level. The IBM documentation, confusingly, states that `level-1` in their grammar is for COBOL levels in the range 2-48: https://www.ibm.com/docs/en/db2-for-zos/13?topic=cobol-host-variable-arrays-in#db2z_hostvariablearraycobol__title__16

Reply:
That is not the correct reference you sent. This is https://www.ibm.com/docs/en/db2-for-zos/13?topic=cobol-host-variables-in#db2z_hostvariablecobol__title__23
It can be 01 for level.

Reply:
I would use the pdf attached in the issue https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/issues/2320